pypesto/hierarchical/inner_calculator_collector.py

```diff
@@ -89,9 +90,9 @@ class InnerCalculatorCollector(AmiciCalculator):
     def __init__(
         self,
         data_types: set[str],
-        petab_problem: "petab.Problem",
+        petab_problem: petab.Problem,
         model: AmiciModel,
-        edatas: list["amici.ExpData"],
+        edatas: list[amici.ExpData],
```

Comment:
I thought that adding " " on petab and amici objects was good for tests not to break if the module is imported, but petab and amici are not installed. I think I tried having it like this, but then tests were failing. Why is it ok to change now?

Reply:
tl;dr: `from __future__ import annotations`

details: https://peps.python.org/pep-0563/